alphabase/io/hdf.py

```diff
@@ -45,7 +45,7 @@ def editing_mode(self, read_only: bool = False, truncate: bool = True):
 
     @property
     def metadata(self):
-        with h5py.File(self.file_name, "a") as hdf_file:
+        with h5py.File(self.file_name) as hdf_file:
```

Comment:
Do I understand it correctly that this is the same as`h5py.File(self.file_name, 'r')` ?
Can we add the `'r'` explicitly so it's clear that this was a deliberate choice.

Reply:
hm, isn't not adding "a" deliberate enough? ;-)